pkg/network/master/metrics/register.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+// contains all controller/master metric definitions and registration
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	metricSDNNamespace           = "sdn"
+	metricSDNSubsystemController = "controller"
+)
+
+var metricEgressIPCount = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: metricSDNNamespace,
+	Subsystem: metricSDNSubsystemController,
+	Name:      "num_egress_ips",
+	Help:      "The number of defined egress IP addresses",
+})
+
+// represents kind EgressNetworkPolicy from API version network.openshift.io/v1
+var metricEgressFirewallRuleCount = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: metricSDNNamespace,
+	Subsystem: metricSDNSubsystemController,
+	Name:      "num_egress_firewall_rules",
+	Help:      "The number of egress firewall rules defined"},
```

Comment:
Can you add a metric for number of distinct ENPs too? I think both would be useful.